### PR TITLE
test for #307

### PR DESCRIPTION
**Pull request by jogibear9988**

### CODE REVIEW

File: Tests/Linq/UserTests/Issue307Tests.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	using Model;
+
+	[TestFixture]
+	public class Issue307Tests : TestBase
+	{
+
+		[Table(Name = "Person")]
+		public class Entity307
+		{
+			private Entity307()
```
Comment (sdanyliv) [inline Tests/Linq/UserTests/Issue307Tests.cs]: @jogibear9988, check that this constructor is called. Put here some informative.

**Comment by jogibear9988** [comment]
Why should it not be called? We have no other!

**Comment by sdanyliv** [comment]
Just for reinsurrance. I'm not near computer, but i think your sample is not clear.

**Comment by sdanyliv** [comment]
@jogibear9988, yes it works, my fault.

**Pull request merged**